### PR TITLE
Add my credentials

### PR DESCRIPTION
**Pull request by jdnavarro**
This is the same as #1015, but using a different branch in my fork instead of `develop`. This way it's easier to keep rebasing the code in this PR if more commits are added to the main `develop` branch.

### CODE REVIEW

File: app/spdb/build.sbt [hunk 1]
```diff
@@ -221,32 +222,36 @@ def swalker(version: Version) = AppConfig(
   )
 ) extending List(with_gogo(version), swalker_credentials(version))
 
-// FNUSSBER
-def fnussber(version: Version) = AppConfig(
```
Comment (cquiroz) [inline app/spdb/build.sbt]: As in the previous PR, I don't think we should delete fnussber configuration until we make sure olesja/sabrina can add their own

Comment (swalker2m) [inline app/spdb/build.sbt]: I guess we could just add Danny's config for now.

**Comment by jdnavarro** [comment]
Now Florian's credentials are still present. Once this is merged I can commit my credentials to the svn repo as well.

**Comment by cquiroz** [comment]
👍

**Comment by cquiroz** [comment]
@jdnavarro We usually wait for the build on travis to complete before a merge, though we don't enforce it on github

**Comment by jdnavarro** [comment]
@cquiroz, sure, I'll make sure it builds completely next time.

**Pull request merged**